setup.py

```diff
@@ -89,6 +89,7 @@ def get_version(filename="jrnl/__init__.py"):
         "keyring>=7.3",
         "passlib>=1.6.2",
         "pyxdg>=0.25",
+        "asteval>=0.9.8",
```

Comment:
Yeah, you're right. It isn't in my checkin, but if you look at this
[checkin](https://github.com/maebert/jrnl/commit/8b64cc95d9e5171b7b914451f8538783b77c2c71), it adds `asteval` in [template.py](https://github.com/maebert/jrnl/commit/8b64cc95d9e5171b7b914451f8538783b77c2c71#diff-46697bd48828cb658c9837ea175fab2fR2).

When the Travis build ran, it was failing, so I added it to fix that dependency.

I didn't know how to handle that. Is there a better way of doing that?